thriftpy/thrift.py

```diff
@@ -176,6 +179,7 @@ def _recv(self, _api):
 
 
 class TProcessor(object):
+
```

Comment:
Sorry for replying late,  in next commit, I will revert this change. However, in Line 296, this blank line is suggested,  before defining the function.